t/lib/OpenQA/Test/Utils.pm

```diff
@@ -656,4 +656,6 @@ sub mock_io_loop (%args) {
     return $io_loop_mock;
 }
 
+sub assume_all_assets_exist { OpenQA::Schema->singleton->resultset('Assets')->search({})->update({size => 0}) }
```

Comment:
I am not so sure about the word "assume" as the result of calling this function is not clear. How about "ensure"?

Reply:
"ensure" makes it sound like it would possibly create assets if they don't exist. This function merely makes our code assume they exist (by putting *some* size into the database).

Reply:
Ok, I get that. I would say that effectively assets do exist for the sake of testing just by adding a defined size :) But ok, let's keep "assume"